spec/jobs/export_user_archive_spec.rb

```diff
@@ -316,10 +316,56 @@ def make_component_json
       expect(DateTime.parse(data[2]['reminder_last_sent_at'])).to eq(DateTime.parse(now.to_s))
       expect(data[2]['reminder_set_at']).to eq('')
 
-      expect(data[3]['topic_id']).to eq(message.id.to_s)
+      expect(data[3]['topic_id']).to eq(private_message_topic.id.to_s)
       expect(data[3]['link']).to eq('')
     end
 
+    context "for polymorphic bookmarks" do
+      let(:component) { 'bookmarks_polymorphic' }
+      before do
+        SiteSetting.use_polymorphic_bookmarks = true
+      end
+
+      it "properly includes bookmark records" do
+        now = freeze_time '2017-03-01 12:00'
+
+        bkmk1 = manager.create_for(bookmarkable_id: post1.id, bookmarkable_type: "Post", name: name)
```

Comment:
```suggestion
        bookmark1 = manager.create_for(bookmarkable_id: post1.id, bookmarkable_type: "Post", name: name)
```

A personal nit of mine xD I would always trade readability over having to type more characters.

Reply:
Hah I was also annoyed by this variable name, will change across the board there :+1: